config/locales/idv/fr.yml

```
@@ -307,9 +308,10 @@ fr:
       technical_difficulties: Malheureusement, nous rencontrons des difficultés
         techniques et ne pouvons pas vérifier votre identité pour le moment.
     warning:
-      attempts:
-        one: Pour des raisons de sécurité, il vous reste une tentative.
-        other: Pour des raisons de sécurité, il vous reste %{count} tentatives.
+      attempts_html:
+        one: Pour des raisons de sécurité, il vous reste <strong>une tentative.</strong>
```

Place period outside strong tags for consistency.
```suggestion
        one: Pour des raisons de sécurité, il vous reste <strong>une tentative</strong>.
```